charts/quantum-serverless/charts/gateway/templates/raycertgen.yaml

```diff
@@ -232,4 +234,54 @@ data:
     kubectl get secret $CLUSTER_NAME -o=jsonpath='{.data.ca\.crt}' | base64 -d > $BASE_DIR/ca.crt
     kubectl get secret $CLUSTER_NAME -o=jsonpath='{.data.tls\.crt}' | base64 -d > $BASE_DIR/tls.crt
     kubectl get secret $CLUSTER_NAME -o=jsonpath='{.data.tls\.key}' | base64 -d > $BASE_DIR/tls.key
+  gencert_cert_worker.sh: |
+    #!/bin/sh
+    BASE_DIR=$1
+    CLUSTER_NAME=$2
+    IP_ADDRESS=$3
+    NAMESPACE=$4
```

Comment:
I'm trying to understand what the pull request is doing Aki, so correct if I'm wrong. `NAMESPACE` argument is not needed in this case, isn't it? At least by the script.